src/Microsoft.Identity.Web/Extensions.cs

```diff
@@ -3,27 +3,27 @@
 
 namespace Microsoft.Identity.Web
 {
-    /// <summary>
-    /// Extension methods.
-    /// </summary>
-    internal static class Extensions
-    {
-        /// <summary>Determines whether the specified string collection contains any.</summary>
-        /// <param name="searchFor">The search for.</param>
-        /// <param name="stringCollection">The string collection.</param>
-        /// <returns>
-        ///   <c>true</c> if the specified string collection contains any; otherwise, <c>false</c>.</returns>
-        public static bool ContainsAny(this string searchFor, params string[] stringCollection)
-        {
-            foreach (string str in stringCollection)
-            {
-                if (searchFor.Contains(str))
-                {
-                    return true;
-                }
-            }
+	/// <summary>
+	/// Extension methods.
+	/// </summary>
+	internal static class Extensions
+	{
+		/// <summary>Determines whether the specified string collection contains any.</summary>
+		/// <param name="searchFor">The search for.</param>
+		/// <param name="stringCollection">The string collection.</param>
+		/// <returns>
+		///   <c>true</c> if the specified string collection contains any; otherwise, <c>false</c>.</returns>
+		public static bool ContainsAny(this string searchFor, params string[] stringCollection)
+		{
+			foreach (string str in stringCollection)
```

Comment:
This can be done faster, but probably shouldn't be. The faster implementation I'm thinking of involves char operations and a trie built from the string collection inputs..